scripts/publish_book.sh

```diff
@@ -0,0 +1,18 @@
+# Exit if any subcommand fails
+set -e
+
+if [ "$CIRCLE_BRANCH" == "develop" ]; then
```

Comment:
@stefandeml think we agreed for this to be master? Otherwise looks good!

Reply:
As far as I see $CIRCLE_BRANCH is the source branch, hence we have to use "develop", no?

Reply:
No I don't think so: after we've merged from a release branch to master, CircleCI builds master which should trigger deployment. Or am I missing something?

Reply:
```suggestion
if [ "$CIRCLE_BRANCH" == "master" ]; then
```